homeassistant/components/kodi/browse_media.py

```diff
@@ -274,6 +295,7 @@ def library_payload(media_library):
         "library_music": "Music",
         MEDIA_TYPE_MOVIE: "Movies",
         MEDIA_TYPE_TVSHOW: "TV shows",
+        MEDIA_TYPE_CHANNEL: "Channels",
```

Comment:
Above we use the title `TV Channels` and here `Channels`. Is the difference intended?

Reply:
I don't believe it matters but will switch to title "Channels" just to be uniform.